openapi_generator_cli/__init__.py

```diff
@@ -18,7 +18,16 @@ def run():
     )
     arguments.append(jar_path)
 
-    if len(sys.argv) > 1:
-        arguments.extend(sys.argv[1:])
+    if args and type(args)==list:
+        arguments.extend(args)
 
     subprocess.call(" ".join(arguments), shell=True)
+
+def cli():
+    args = []
+    if len(sys.argv) > 1:
+        args = sys.argv[1:]
+    run(args)
+
+if __name__ == "__main__":
+    cli()
```

Comment:
You should follow black correction.

https://github.com/OpenAPITools/openapi-generator-pip/blob/91bc8433f974e06fc1201bc23658e2fe8dfab127/README.md?plain=1#L4

```suggestion
    if args and type(args) == list:
        arguments.extend(args)

    subprocess.call(" ".join(arguments), shell=True)


def cli():
    args = []
    if len(sys.argv) > 1:
        args = sys.argv[1:]
    run(args)


if __name__ == "__main__":
    cli()

```